MAUI/ListView/working-with-sflistview.md

```diff
@@ -122,7 +122,9 @@ The [ItemLongPress](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView
 
 * To show the context menu.
 
-N> Interaction events of the `SfListView` will be triggered for all the `ListViewItems` like `HeaderItem`, `FooterItem`, `GroupHeaderItem`, `LoadMoreItem`, and `RecordItem`. You can handle the interaction actions based on the `ItemType` of the event args. 
+N>
+* Interaction events of the `SfListView` will be triggered for all the `ListViewItems` like `HeaderItem`, `FooterItem`, `GroupHeaderItem`, `LoadMoreItem`, and `RecordItem`. You can handle the interaction actions based on the `ItemType` of the event args. 

```

Comment:
* Interaction events of the `SfListView` will be triggered for all the `ListViewItems` like `HeaderItem,` `FooterItem,` `GroupHeaderItem,` `LoadMoreItem,` and `RecordItem.` Handle the interaction actions based on the `ItemType` of the event args.

Reply:
Changes committed